tests/ManageCourses.Tests/DbIntegration/EnrichmentServiceTests.cs

```diff
@@ -157,6 +163,20 @@ public void Test_InstitutionEnrichment_workflow_should_not_error()
             var updateResult = enrichmentService.GetInstitutionEnrichment(_providerInstCode, _email);
             updateResult.EnrichmentModel.TrainWithDisability.Should().BeEquivalentTo(trainWithDisabilityUpdatedText);
             updateResult.EnrichmentModel.TrainWithUs.Should().BeEquivalentTo(trainWithUsUpdatedText);
+            updateResult.LastPublishedTimestampUtc.Should().BeNull();
+            //publish
+            var publishResults = enrichmentService.PublishInstitutionEnrichment(_providerInstCode.ToLower(), _email);
+            publishResults.Should().BeTrue();
+            var publishRecord = enrichmentService.GetInstitutionEnrichment(_providerInstCode.ToLower(), _email);
+            publishRecord.Status.Should().BeEquivalentTo(EnumStatus.Published);
+            publishRecord.LastPublishedTimestampUtc.Should().NotBeNull();
+            //test save again after publish
+            enrichmentService.SaveInstitutionEnrichment(model, _providerInstCode.ToLower(), _email);
+            var updateResult2 = enrichmentService.GetInstitutionEnrichment(_providerInstCode, _email);
+            updateResult2.EnrichmentModel.TrainWithDisability.Should().BeEquivalentTo(trainWithDisabilityText);
+            updateResult2.EnrichmentModel.TrainWithUs.Should().BeEquivalentTo(trainWithUsText);
+            updateResult2.Status.Should().BeEquivalentTo(EnumStatus.Draft);
+            updateResult2.LastPublishedTimestampUtc.ToString().Should().BeEquivalentTo(publishRecord.LastPublishedTimestampUtc.ToString());
```

Comment:
Might be good to count the number of records of each type that ended up in the db at this point,